File: Tarea05/ejercicio3.c
#include <stdio.h>
#include <math.h>

int main() {
	float a;
	float b;
	float c;
	float raiz1;
	float raiz2;
	float dis;
	printf("Introduce el coeficiente de x²\n");
	scanf("%f", &a);
	printf("Introduce el coeficiente de x¹\n");
	scanf("%f", &b);
	printf("Introduce el coeficiente de x⁰\n");
	scanf("%f", &c);
	dis = b*b - 4*a*c;
	if (dis > 0){
		raiz1 = (-b + (sqrt(dis))/2*a);
		raiz2 = (-b - (sqrt(dis))/2*a);
		printf("Las raices de la ecuacion son: %f, %f\n", raiz1, raiz2);
	}
	return 0;
}

File: Tarea01/ejercicio3.py
print("Introduce el coeficiente de x²")
a = int(input())
print("Introduce el coeficiente de x¹")
b = int(input())
print("Introduce el coeficiente de x⁰")
c = int(input())
if (b**2)-4*a*c > 0:
   raiz1 = (-b + ((b**2) - 4*a*c)**(1/2))/(2*a)
   raiz2 = (-b - ((b**2) - 4*a*c)**(1/2))/(2*a)
   print("Las raíces a la ecuación son: %f, %f"%(raiz1, raiz2))
else:
   print("La ecuación no tiene raíces reales")
